include/json/writer.h

```diff
@@ -270,6 +275,8 @@ class JSONCPP_DEPRECATED("Use StreamWriterBuilder instead") JSON_API StyledWrite
  * \sa Reader, Value, Value::setComment()
  * \deprecated Use StreamWriterBuilder.
  */
+#pragma warning(push)
+#pragma warning(disable:4996) // Deriving from deprecated class
```

Comment:
I don't actually think that disabling the deprecation warnings on this class are necessary. They were added to `StyledWriter` and `FastWriter` since both those files derived from `Writer` which is deprecated (and thus considered by VS to be using `Writer`). `StyledStreamWriter` is itself deprecated (same as `Writer`) and doesn't  use a deprecated class.

I know I referred to the stackoverflow comment but I didn't examine the class it referred to. Now I think it's in error. (Although I don't think it's harmful either, I can't really tell since I'm not at my dev machine).

Reply:
Ok. I'll remove the push/pop for StyledStreamWriter. Thanks for following up.